### PR TITLE
MTM-49511 revoke tokens documentation

### PR DESCRIPTION
**Pull request by mmed-softwareag**

### CODE REVIEW

**Review by Michal-Bak — APPROVED** [review]

File: content/users-guide/administration-bundle/managing-users.md [hunk 1]
```diff
@@ -180,3 +180,11 @@ Click the menu icon at the right of the respective row and then click **Delete**
 {{< c8y-admon-info >}}
 You need a role with user management permission to perform this option.
 {{< /c8y-admon-info >}}
+
+### To revoke user tokens
```
Comment (vfo-softwareag) [inline content/users-guide/administration-bundle/managing-users.md:184]:
```suggestion
### Revocation of all user tokens
```

File: content/users-guide/administration-bundle/managing-users.md [hunk 2]
```diff
@@ -180,3 +180,11 @@ Click the menu icon at the right of the respective row and then click **Delete**
...
+
+Click **Revoke tokens** button on the right of the top menu bar. Revoking tokens will log out all users logged with "OAI-Secure" or "Single sign-on redirect". Note that also JWT tokens retrieved by device will be revoked.
```
Comment (rssag) [inline content/users-guide/administration-bundle/managing-users.md:186]:
```suggestion
Click the **Revoke tokens** button on the right of the top menu bar. Revoking tokens logs out all users currently logged in via OAI-Secure or Single sign-on redirect. Note that JWT tokens retrieved by devices are also revoked.
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 3]
```diff
@@ -47,6 +47,13 @@ The **User** menu contains the following items:
 Note that this option is only available if support user access is not set globally for subtenant users in the {{< management-tenant >}}, for details see <a href="/users-guide/enterprise-tenant/#support-user-access" class="no-ajaxy">Support user access</a>. After the support user access has been activated, the menu item switches to <strong>Deactivate support</strong>. If your support request has been resolved but the duration for the support user access is not expired (24 hours per default) you can actively disable an active support user request here.</td>
 </tr>
 <tr>
+<td style="text-align:left">Revoke tokens</td>
+<td style="text-align:left">Revoking tokens will log out all users logged with "OAI-Secure" or "Single sign-on redirect". Note that also JWT tokens retrieved by device will be revoked.<br>
```
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:51]:
```suggestion
<td style="text-align:left">Revoking tokens logs out all users currently logged in via OAI-Secure or single sign-on redirect. Note that JWT tokens retrieved by devices are also revoked.<br>
```

Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:51]:
```suggestion
<td style="text-align:left">Revoking tokens will log out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that the JWT tokens retrieved by a device will be revoked as well.<br>
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 4]
```diff
@@ -47,6 +47,13 @@ The **User** menu contains the following items:
...
+<br>
+To revoke tokens, the user must have the permission ROLE_USER_MANAGEMENT_ADMIN.
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:53]:
```suggestion
To revoke tokens, the user must have ADMIN permission for the permission type "User management".  
```

File: content/users-guide/administration-bundle/managing-users.md [hunk 5]
```diff
@@ -180,3 +180,11 @@ Click the menu icon at the right of the respective row and then click **Delete**
...
+
+{{< c8y-admon-info >}}
+To revoke tokens, you must have the permission ROLE_USER_MANAGEMENT_ADMIN.
```
Comment (rssag) [inline content/users-guide/administration-bundle/managing-users.md:189]:
```suggestion
To revoke tokens, you need ROLE_USER_MANAGEMENT_ADMIN permission.
```

File: content/users-guide/getting-started-bundle/user-settings.md (hunk 4 above)
Comment (rssag) [inline content/users-guide/getting-started-bundle/user-settings.md:53]:
```suggestion
To revoke tokens, you need ROLE_USER_MANAGEMENT_ADMIN permission.
```

File: content/users-guide/administration-bundle/managing-users.md [hunk 6]
```diff
@@ -180,3 +180,11 @@ Click the menu icon at the right of the respective row and then click **Delete**
 {{< c8y-admon-info >}}
 You need a role with user management permission to perform this option.
 {{< /c8y-admon-info >}}
+
+### To revoke user tokens
+
+Click **Revoke tokens** button on the right of the top menu bar. Revoking tokens will log out all users logged with "OAI-Secure" or "Single sign-on redirect". Note that also JWT tokens retrieved by device will be revoked.
+
+{{< c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:188]:
```suggestion
{{< c8y-admon-req >}}
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/users-guide/administration-bundle/managing-users.md (hunk 5 above)
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:189]:
```suggestion
To revoke tokens, the user must have ADMIN permission for the permission type "User management".
```

**Comment by rssag** [comment]
which releases does this apply to? @mmed-softwareag

File: content/users-guide/administration-bundle/managing-users.md [hunk 7]
```diff
@@ -180,3 +180,11 @@ Click the menu icon at the right of the respective row and then click **Delete**
...
+To revoke tokens, you must have the permission ROLE_USER_MANAGEMENT_ADMIN.
+{{< /c8y-admon-info >}}
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:190]:
```suggestion
{{< /c8y-admon-req >}}
```

File: content/users-guide/administration-bundle/managing-users.md (hunk 2 above)
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:186]:
```suggestion
Click **Revoke tokens** at the right of the top menu bar. Revoking tokens will log out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that the JWT tokens retrieved by a device will be revoked as well.
```

File: content/users-guide/administration-bundle/managing-users.md (hunk 1 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-users.md:184]: @vfo-softwareag Instruction headings always start with "To..." Should stay as is.

Comment (vfo-softwareag) [reply content/users-guide/administration-bundle/managing-users.md:184]: @BeateRixen okay. I see. I didn't know that. Sorry.

File: content/users-guide/administration-bundle/managing-users.md (hunk 5 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-users.md:191]: This is not the way we refer to roles and permissions in the UI context, @rssag :-). This is the dev terminology.

File: content/users-guide/administration-bundle/managing-users.md [hunk 8]
```diff
@@ -180,3 +180,13 @@ Click the menu icon at the right of the respective row and then click **Delete**
 {{< c8y-admon-info >}}
 You need a role with user management permission to perform this option.
 {{< /c8y-admon-info >}}
+
+### To revoke user tokens
+
+In the event of a security incident involving the session tokens of your tenant's users, you can invalidate any tokens currently in use.
+
+To revoke all session tokens click **Revoke tokens**  at the right of the top menu bar. Revoking all tokens logs out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:188]:
```suggestion
To revoke all session tokens click **Revoke tokens**  at the right of the top menu bar. Revoking all tokens logs out all users logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.
```

File: content/users-guide/administration-bundle/managing-users.md [hunk 9]
```diff
@@ -180,3 +180,13 @@ Click the menu icon at the right of the respective row and then click **Delete**
...
+
+{{< c8y-admon-req >}}
+To revoke tokens, the user must have ADMIN permission for the permission type "User management".
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:191]:
```suggestion
To revoke tokens, you must have ADMIN permission for the permission type "User management".
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 10]
```diff
@@ -47,6 +47,13 @@ The **User** menu contains the following items:
 Note that this option is only available if support user access is not set globally for subtenant users in the {{< management-tenant >}}, for details see <a href="/users-guide/enterprise-tenant/#support-user-access" class="no-ajaxy">Support user access</a>. After the support user access has been activated, the menu item switches to <strong>Deactivate support</strong>. If your support request has been resolved but the duration for the support user access is not expired (24 hours per default) you can actively disable an active support user request here.</td>
 </tr>
 <tr>
+<td style="text-align:left">Revoke tokens</td>
+<td style="text-align:left">Revoking tokens will log out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.<br>
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:51]:
```suggestion
<td style="text-align:left">Revoking tokens logs out all users currently logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.<br>
```

File: content/users-guide/getting-started-bundle/user-settings.md [hunk 11]
```diff
@@ -47,6 +47,13 @@ The **User** menu contains the following items:
...
+<br>
+To revoke tokens, the user must have ADMIN permission for the permission type "User management".
```
Comment (BeateRixen) [inline content/users-guide/getting-started-bundle/user-settings.md:53]:
```suggestion
To revoke tokens, you must have ADMIN permission for the permission type "User management".  
```

**Review by BeateRixen — APPROVED** [review]

**Review by vfo-softwareag — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**